ee/cis/macos-13/cis-policy-queries.yml

```diff
@@ -1733,3 +1733,44 @@ spec:
   purpose: Informational
   tags: compliance, CIS, CIS_Level1, CIS6.1.1
   contributors: artemist-work
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: CIS - Ensure Automatic Opening of Safe Files in Safari Is Disabled
+  platforms: macOS
+  platform: darwin
+  description: |
+    Safari will automatically run or execute what it considers safe files. This can include
+    installers and other files that execute on the operating system. Safari evaluates file safety by
+    using a list of filetypes maintained by Apple. The list of files include text, image, video and
+    archive formats that would be run in the context of the OS rather than the browser.
+    Hackers have taken advantage of this setting via drive-by attacks. These attacks occur when a
+    user visits a legitimate website that has been corrupted. The user unknowingly downloads a
+    malicious file either by closing an infected pop-up or hovering over a malicious banner. An
+    attacker can create a malicious file that will fall within Safari's safe file list that will
+    download and execute without user input.
+  resolution: |
+    Graphical Method:
+    1. Open Safari
+    2. Select Safari > Settings from the menu bar
+    3. Select General
+    4. Set Open safe files after downloading to disabled
+    Payload Method:
+    Ask your administrator to deploy a profile which disables AutoOpenSafeDownloads on Safari
+  query: |
+    SELECT 1 WHERE NOT EXISTS (
+      SELECT 1 FROM plist WHERE
+        path LIKE
+        '/Users/%/Library/Containers/com.apple.Safari/Data/Library/Preferences/com.apple.Safari.plist' AND
+        key = 'AutoOpenSafeDownloads' AND
+        value = '1'
+    ) OR EXISTS (
+      SELECT 1 FROM managed_policies WHERE
+        domain = 'com.apple.Safari' AND
+        name = 'AutoOpenSafeDownloads' AND
+        value = '0'
+    );
```

Comment:
Given this note (and that we have a test profile) I'd stick with the `managed_policies` part of the query:
```
Note: Since the profile method sets a system-wide setting and not a user-level one, the profile method is the preferred method. It is always better to set system-wide than per user.
```

Reply:
Okay, I'll do MDM only. I've been trying to avoid it but I guess anyone who cares about CIS will be using MDM anyway.